requirements.txt

```diff
@@ -7,3 +7,6 @@
 aioredis==1.2.0
 async-timeout==3.0.1      # via aioredis
 hiredis==1.0.0            # via aioredis
+mypy==0.730
+black==19.3b0
```

Comment:
1) This file is autogenerated. You should modify requirements.in instead (adding `make requirements` running pip-compile would be a good addition)
2) These are runtime requirements. Linting tools should not be there.

Linting requirements could go into requirements-dev for now